awx/main/management/commands/inventory_import.py

```diff
@@ -577,16 +595,29 @@ def _delete_group_children_and_hosts(self):
             for offset in range(0, len(del_host_pks), self._batch_size):
                 del_pks = del_host_pks[offset:(offset + self._batch_size)]
                 for db_host in db_hosts.filter(pk__in=del_pks):
-                    group_host_count += 1
                     if db_host not in db_group.hosts.all():
                         continue
+                    group_host_removals += 1
                     db_group.hosts.remove(db_host)
                     logger.debug('Host "%s" removed from group "%s"',
                                  db_host.name, db_group.name)
+        logger.info('Source return {} group children memberships versus {} existing'.format(
+            group_group_count, prior_group_group_count
+        ))
+        if group_group_removals:
+            logger.warning('Removed {} group children memberships'.format(group_group_removals))
+        else:
+            logger.info('No group children were removed.')
+        logger.info('Source returned {} group host memberships versus {} existing'.format(
+            group_host_count, prior_group_host_count
+        ))
+        if group_host_removals:
+            logger.warning('Removed {} group host memberships'.format(group_host_removals))
+        else:
+            logger.info('No group hosts were removed')
         if settings.SQL_DEBUG:
-            logger.warning('group-group and group-host deletions took %d queries for %d relationships',
```

Comment:
So, here, we conditionally reported (depending on dev/production) the number of group-children removals, plus the number of group-host relationships _inspected_ (not the number removed). So this is not a count, either of "relationships", or of "removals"... it's just... a number.